### ISSUE TITLE
New feature: Attribute tooltips

### ISSUE THREAD

**Issue by mMoliere** [majlis-erc/manuForma#150]

**Comment by mMoliere** [comment]
@wsalesky For some of the attribute values we have, we would like the option to display explanations. I think they should work analogous to the tool tips we have for the elements, with little windows. The difference being that the attribute values won't get little symbols for the mouse to hover over. Instead, mouse hovering over the name should trigger the tool tip. 

I have added `desc` elements to some `valItem` elements in mss-definition.xml schema. And if you can make this work, I would add them to other parts of the schema as well.

**Comment by wsalesky** [comment]
@mMoliere As far as I can see, there is no way to add these to the select options with the XForms technology. We can leave this open while I do more testing, but I think it is not going to be a viable option.

**Comment by mMoliere** [comment]
@wsalesky Fair enough.

**Comment by mMoliere** [comment]
@wsalesky How about the attribute headlines get their own tooltip buttons?

Like this
![Screenshot 2023-07-25 at 12 55 58](https://github.com/majlis-erc/manuForma/assets/73166992/7edef2b8-90fd-457a-97cc-b949e1aab1e9)

**Comment by wsalesky** [comment]
@mMoliere Yes, that should be possible, go ahead and add it to the schema and I will get it into the forms.

**Comment by wsalesky** [comment]
@mMoliere did this ever get added to the schema? (Just trying to catch up on all the lose ends.)
